status/module.go

```diff
@@ -1,66 +1,36 @@
 package status
 
 import (
-	"github.com/safing/portbase/database"
-	"github.com/safing/portbase/log"
+	"context"
+
 	"github.com/safing/portbase/modules"
+	"github.com/safing/portmaster/netenv"
 )
 
 var (
 	module *modules.Module
 )
 
 func init() {
-	module = modules.Register("status", nil, start, stop, "base")
+	module = modules.Register("status", nil, start, nil, "base")
 }
 
 func start() error {
-	err := initSystemStatus()
-	if err != nil {
-		return err
-	}
-
-	err = startNetEnvHooking()
+	module.StartWorker("auto-pilot", autoPilot)
+	triggerAutopilot()
+
+	err := module.RegisterEventHook(
+		"netenv",
+		netenv.OnlineStatusChangedEvent,
+		"update online status in system status",
+		func(_ context.Context, _ interface{}) error {
+			triggerAutopilot()
+			return nil
+		},
+	)
```

Comment:
Why is the autopilot triggered when the network changes?

Reply:
It won't do much here as the selected security level has not yet changed but it will push a new `SystemStatusRecord`.  Though I decided to trigger the whole autopilot since we will add "untrusted" network detection anyway.

Reply:
The network trust system will be using Threats anyway, so that will be taken care of. But yes, we need to update for the online status.